fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java

```diff
@@ -87,8 +87,11 @@ public final class FeMetaVersion {
     // For transaction insert
     public static final int VERSION_132 = 132;
 
+    // support db comment
+    public static final int VERSION_133 = 133;
+
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_132;
+    public static final int VERSION_CURRENT = VERSION_133;
```

Comment:
reabase to newest master, version_current is version_133, u need to use version_134

Reply:
Yes, I've modified it. Thanks.